Source/WebKit/UIProcess/API/Cocoa/WKBackForwardListItem.mm

```diff
@@ -48,6 +48,7 @@ - (NSURL *)URL
     return [NSURL _web_URLWithWTFString:_item->url()];
 }
 
+// This property is deprecated and now always returns the empty string.
 - (NSString *)title
 {
     if (!_item->title())
```

Comment:
* No need for the deprecation comment
 * Please directly return an empty string instead of returning `_item->title()`